client/src/components/SectionTable/SectionTableBody.js

```diff
@@ -171,60 +156,27 @@ const SectionDetailsCell = withStyles(styles)((props) => {
 const InstructorsCell = withStyles(styles)((props) => {
     const { classes, instructors } = props;
 
-    const CustomTooltip = withStyles((theme) => ({
-        tooltip: {
-            backgroundColor: theme.palette.background.paper,
-            color: theme.palette.text.primary,
-        },
-    }))(Tooltip);
-
-    const handleClick = (isRMP, profName) => {
-        const lastName = profName.substring(0, profName.indexOf(','));
-
-        if (!isRMP) {
-            window.open(`https://eaterevals.eee.uci.edu/browse/instructor#${lastName}`);
-            ReactGA.event({
-                category: 'antalmanac-rewrite',
-                action: `Click instructor name`,
-                label: `EaterEvals`,
-            });
-        } else {
-            window.open(
-                `https://www.ratemyprofessors.com/search.jsp?queryBy=teacherName&schoolName=university+of+california+irvine&queryoption=HEADER&query=${lastName}&facetSearch=true`
-            );
-            ReactGA.event({
-                category: 'antalmanac-rewrite',
-                action: `Click instructor name`,
-                label: `RateMyProfessors`,
-            });
-        }
-    };
-
-    const DualButton = (props) => {
-        return (
-            <ButtonGroup size="small">
-                <Button key={1} value="left" onClick={() => handleClick(false, props.profName)}>
-                    EaterEvals
-                </Button>
-                <Button key={2} value="right" onClick={() => handleClick(true, props.profName)}>
-                    RMP
-                </Button>
-            </ButtonGroup>
-        );
-    };
-
     const getLinks = (professorNames) => {
         return professorNames.map((profName) => {
             if (profName !== 'STAFF') {
+                const lastName = profName.substring(0, profName.indexOf(','));
                 return (
-                    <CustomTooltip
-                        key={profName}
-                        interactive
-                        placement="left"
-                        title={<DualButton profName={profName} />}
-                    >
-                        <div className={classes.link}>{profName}</div>
-                    </CustomTooltip>
+                    <div>
+                        <a
+                            href={`https://www.ratemyprofessors.com/search/teachers?sid=U2Nob29sLTEwNzQ=&query=${lastName}`}
```

Comment:
Updated the URL to include UCI's school id. This provides better search results than the previous url.